travis_ci/check-validations.py

```diff
@@ -0,0 +1,47 @@
+#! python
+
+import logging
+import os
```

Comment:
```python
import os.path
```
code as you wrote works, but in python we prefer to import the path subpackage explicitely